### DISCUSSION TITLE
Trading Cards

### DISCUSSION THREAD

**Discussion by mikekasprzak** [ludumdare/ludumdare#2004] [Ideas]
Let people list what their skills are. Skills are grouped in 
- Programming
- Graphics
- Audio
- Design
- Admin

(all lists subject to change)
### Programming
- Web
- Middleware
- Low Level
- 2D
- 3D
- Other smarty topics
### Graphics
- 2D
- 3D
- Illustration
- Animation
- Concept
- Trailers/Video
### Audio
- Music
- Sound Design
- Voice Over/Acting
- Foley (?)
- Engineering (i.e. recording people)
### Design
- Game Design
- Level Design
- Something that means tuning
### Writing (note, consider merging with design)
- Writing ... uh
### Administration
- Leadership
- Assistance
- Quality Assurance

(**NOTE**: The above lists are just guesses, roughly what and where things go when describing a skillset. Actual research is going to have to happen when building categories.)

Anyway, the point is to let people summarize their contribution to games.

People should be able to select their 'Forte', and possibly their 'Preference'. Once we know more about a person's skill (either via participated events, or fellow users giving "Love" to their art skill or such), we can begin to validate someone's skill. Though that sounds fancy, it'll just mean we can back up the claims that a person is good at something (because we have the evidence).
# The Point

The point of this: When you hover over an an avatar, you get a popup that tells you about the person. Easly see if they're a programmer, artist, musician, skilled in multiple disciplines, etc. When they comment on certain subjects, it's there to help validate how much of an authority they are on a subject.

Ludum Dare participation wont be the only way to get validated, but it's the recommended way.
# Challenges

Though conceptually simple, this potentially has ramifications beyond Ludum Dare, especially for those looking to get hired. Care should be taken so that validation can't be abused, and that people unfamiliar with Ludum Dare can easily understand what it means. Like we've seen with other Ludum Dare things, it's only a matter of time before someone puts this on their resume.

How this works with `jammer.bio` is very important.

**Comment by mikekasprzak** [comment]
Started a survey to help get some real data.

http://goo.gl/forms/HB8th4UVwC

**Comment by mikekasprzak** [comment]
Initial feedback seems that a surprising number of people identify as Unity Programmers. 

Starting to think this should maybe be more like traits. Each has an icon associated with it. You then build a "trading card" out of those traits.

![tradingcard](https://cloud.githubusercontent.com/assets/1570313/14483685/7d116a02-0117-11e6-80a6-4d60ce2cc3f6.png)

Rough example, but something like this. Large square avatar, your name, 4 traits (Unity Coder, Pixel Artist, whatever, whatever), then 2 numeric stats. Not sure what the numerics should be, it might be adjustable. I.e. number of jam games, ludum dare games, commercial games, karma, etc. Whatever metrics we decide are meaningful.

The "trading cards" are what you get when you hover over someone's avatar.

We could have some implicit styles to the cards that mean something. Like a gold (foil) border for anyone that has ever won a Ludum Dare. Gold Silver Bronze in any category (except coolness). Plus other styles denoting other attributes (popular, prolific, etc)

**Comment by mikekasprzak** [comment]
Current plan for this: There's a stub that lets you click on avatars to "quick info" about a user... it doesn't give you any info, but that's what these are for.

It's currently using a dialog, but ideally it probably should be a floater relative the original avatar icon. I include things, mainly, a follow button.

Undecided if this should support hover. I want to be able to access this via mobile (where we can't hover).

**Comment by philomory** [comment]
On the hover issue, I'd suggest supporting hover, but also click. Supporting hover is a great way for people to discover functionality like this. Once people know about it, they'll still be able to reach it on mobile via a click.
